docs/use_case/post_demo_app_intro.md

```diff
@@ -0,0 +1,53 @@
+In this tutorial we are going to present 3 versions of ONEm's #post application. The apps are written using the Django framework ONEm's Python SDK.
```

Comment:
“the Django framework ONEm’s Python SDK”?
By this you mean "The apps are written in Django framework using ONEm's Python SDK." ?